### PR TITLE
allow configuration of youtube playerVars

### PR DESCRIPTION
**Pull request by mziwisky**

### CODE REVIEW

File: src/js/me-shim.js [hunk 1]
```diff
@@ -841,7 +844,7 @@ mejs.YouTubeApi = {
 			height: settings.height,
 			width: settings.width,
 			videoId: settings.videoId,
-			playerVars: {controls:0, wmode:'transparent'},
```
Comment (a-b-r-o-w-n) [inline src/js/me-shim.js:844]: Does `wmode` need to go in the default options?

**Review by a-b-r-o-w-n — COMMENTED** [review]
I have 1 question.

File: src/js/me-shim.js (hunk 1 above)
Comment (a-b-r-o-w-n) [reply src/js/me-shim.js:844]: nvm. I just check out the docs and didn't see that option. Carry on.

**Review by a-b-r-o-w-n — APPROVED** [review]

File: src/js/me-shim.js (hunk 1 above)
Comment (mziwisky) [reply src/js/me-shim.js:844]: yep, as you discovered, it's obsolete

**Pull request merged**